utils/aarch64_32_bit_el1_supported.c

```diff
@@ -0,0 +1,35 @@
+#include <errno.h>	/* errno for perror() */
+#include <fcntl.h>	/* open() */
+#include <stdio.h>	/* perror() */
+#include <sys/ioctl.h>	/* _IO(), ioctl() */
+#include <unistd.h>	/* close() */
+
+/* From include/uapi/linux/kvm.h */
+#define KVM_CHECK_EXTENSION	_IO(0xAE, 0x03)
+#define KVM_CAP_ARM_EL1_32BIT	93
```

Comment:
Can we just include this header rather than defining the values ourselves?
```c
#include <linux/kvm.h>
```

Reply:
Yes, done. Thanks a lot for the review!